Newsletter: What engineers get wrong about communication

ivanagas opened the pull request:
## Changes

Rough-ish draft

Would love high level feedback for now:
- What's good
- What's bad
- What's missing

## Checklist

- [x] Words are spelled using American English
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [x] Use relative URLs for internal links

## Article checklist

- [ ] I've added (at least) 3-5 internal links to this new article
- [ ] I've added keywords for this page to the rank tracker in Ahrefs
- [ ] I've checked the preview build of the article
- [ ] The date on the article is today's date
- [ ] I've added this to the relevant "Tutorials and guides" docs page (if applicable)

andyvan-ph reviewed (commented):
## What's good?

- I like the focus on user-centric comms right at the start. We could probably go harder on that point.

- I like the framing of we're sharing how we work and how it's not about frameworks etc. All the core points are good, though I think it's a little thin on specifics in some places – see inline comments.

## What's the bad?

Not so much bad, but I wonder if a Dos and Don'ts-style format might give this a bit more edge? 

Atm, it's mostly about about what you should do and it's all pretty uncontroversial, but often the things we don't like / you should avoid are more interesting.

Like, this line really slaps: 

> "The opposite of this is just saying “this isn’t working” or “what do you think of this?” Low context communication like this puts all of the work onto the receiver, leaves more room for ambiguity, and is simply less effective.

But it's relegated to a footnote to the section.

The closest example of what I mean is [one Charles did ages ago](https://posthog.com/founders/stop-treating-engineers-differently), where we did it with alternating truths and myths.

Would something like this but it's Dos and Don'ts work?  

(Mental note: that's a good article we could update for the newsletter!)

Alternatively, maybe the format could "Avoid XYZ" where XYZ is a common bad communication habit, then we outline "Do this instead"?

## What's missing?

- And ending, but I assume that's because it's a draft.

ivanagas commented:
Alright updated to "Communication mistakes for engineers to avoid" to try to give it a bit more ZING + addressed your points 

Still haven't figured out the conclusion though...

daniloc commented:
A thought on concluding:

if the headline frame is "here's what you're avoiding" then the conclusion could be framed affirmatively: here's what you become for avoiding such things.

Clear the table – things to avoid – then set the table with a picture of what you become?

andyvan-ph commented:
Some more random thoughts:

- I feel like we could do with some really useful / good tips on written communication. Currently this has lots of good principles, but I'm not sure what someone would takeaway and say "I'll try doing this". Lior did this well in the issue about feedback: https://newsletter.posthog.com/p/why-you-suck-at-giving-feedback

- As an example, could we pull together some hypothetical or real examples of "here's some bad communication" and "here's a better version" that contains the same information, but organized in a better way with all the context etc. someone would need to take action.

- I'm always a bit suspicious of articles with 5 points. What other things could we highlight here? Random ideas:
  - Not being opinionated... I think there's something in this about how non-opinionated comms creates inertia. Can talking about "strong opinions, weakly held" here.
  - Lack of honesty about feelings... if something is bugging you, you should get it out rather than holding onto it
  - Not communicating urgency... e.g. I need this right now vs not so urgent but still important, etc.

ivanagas commented:
Beginning to see the light at the end of the tunnel of this one :)
- Filled a bunch of points out with more details and examples
- Added a point on "Vibe communicating (AKA lacking an opinion)"
- Pie chart, new meme

Could a hedgehog-ified version of this image be our cover photo?

![squirrel](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_03_11_at_11_29_13_2x_d5fd501927.png)

ivanagas commented on contents/newsletter/how-to-communicate.md:
```diff
@@ -0,0 +1,158 @@
+
+2. **Feedback.** A common sentence we see at PostHog is “X gave feedback that…” followed by a potential solution for that feedback.
+
+3. **Experience.** We care about what the end user experience actually is. Examples and anecdotes from users are hugely valuable.
```
I just removed it, I'm not sure it is really different than feedback

ivanagas commented:
Art requested: https://github.com/PostHog/posthog.com/issues/10972

Need to ask for help from Cory on graphics as well :)

andyvan-ph commented on contents/newsletter/how-to-communicate.md:
```diff
@@ -0,0 +1,182 @@
+
+Being opinionated enables us to move faster and more autonomously. Teams don't wait to be assigned tasks or told what to do. They identify problems and go solve them.
+
+> **Bonus point: Don't hedge.** Once you've done the work to form an opinion, be confident and direct in sharing it. Don't hedge by saying "maybe" or "I'm not sure." Vague, non-committal communication might feel safer, but it rarely leads to action.
```
(the hedging point, I mean)

andyvan-ph commented on contents/newsletter/communication-mistakes.md:
```diff
@@ -0,0 +1,202 @@
+
+It’s easy to get caught up in the technical details of a project, such as overcoming constraints, optimizing performance, and following best practices.  
+
+But companies don’t succeed based on their ability to solve [technical problems](https://newsletter.posthog.com/p/how-we-choose-technologies). They win when they solve their user’s [most painful and valuable problems](https://newsletter.posthog.com/p/how-to-uncover-your-users-real-problems).  
```
I've probably missed this in the, but we should link to the website versions of newsletters on our website.

ivanagas replied:
got it

andyvan-ph commented on contents/newsletter/communication-mistakes.md:
```diff
@@ -0,0 +1,202 @@
+- 97% of people said leaders at PostHog keep people informed about what is happening.  
+- 98% of people said they understand PostHog's goals and can see how their work contributes to them.
+
+Although communication can always be improved, avoiding these mistakes goes a long way in doing it as well as possible.
```
```suggestion
Although communication can always be improved, avoiding these mistakes goes a long way in doing it as well as possible.

<NewsletterForm />
```
